aaad/marathon.py

```diff
@@ -120,15 +120,17 @@ def filterGroups(self, groups_list, allowed_namespaces):
 
         return allowed_groups
 
-    def is_delete_request(self, request_uri):
-        is_delete_request = False
+    def is_quota_validation_required(self, request_uri):
+        is_quota_validation_required = True
         try:
-            uri_pattern = re.compile("^{}$".format("/marathon/v2/tasks/(delete|delete\?.*)"))
-            uri_match = uri_pattern.match(request_uri)
-            if uri_match:
-                is_delete_request = True
-
-            return is_delete_request
+            delete_request_uri_pattern = re.compile("^{}$".format("/marathon/v2/tasks/(delete|delete\?.*)"))
```

Comment:
Can we just do this - 
```
if (request_uri.startswith('/marathon/v2/eventSubscriptions') or request_uri.startswith('/marathon/v2/delete'))
  return false
return true
```
Less expensive and doesn't have to check both conditions.